### PR TITLE
[Travis] Added missing Behat tests to Travis

### PR DESCRIPTION
**Pull request by pamil**
There were 4 suites missing: `i18n`, `channels`, `search_orm_only` and `contact`. Added only three of them, because `contact` is currently under refactoring by @GSadee in #3164.

There were also 3 feature missing in `checkout` suite: `features/frontend/cart_taxation.feature`, `cart_tax_categories.feature` and `features/frontend/cart_inclusive_tax.feature`.

### CODE REVIEW

File: src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php [hunk 1]
```diff
@@ -28,6 +28,11 @@ public function thereArePaymentMethods(TableNode $table)
         $repository = $this->getRepository('payment_method');
 
         foreach ($table->getHash() as $data) {
+            if (!isset($data['calculator']) && !isset($data['calculator_configuration'])) {
```
Comment (stloyd) [inline src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php]:
```php
if (!isset($data['calculator'], $data['calculator_configuration'])) {}

File: src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php [hunk 2]
```diff
@@ -36,7 +41,6 @@ public function thereArePaymentMethods(TableNode $table)
             $method->setFeeCalculatorConfiguration($this->getConfiguration($data['calculator_configuration']));
 
             $enabled = true;
```
Comment (stloyd) [inline src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php]:
```php
$enabled = isset($data['enabled']) ? 'yes' === trim($data['enabled']) : true;

File: src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php (hunk 1 above)
Comment (pamil) [inline src/Sylius/Bundle/PaymentBundle/Behat/PaymentContext.php]: :+1:

**Comment by pjedrzejewski** [comment]
Awesome, thanks a lot Kamil! :+1:

**Pull request merged**